mod-email/src/main/resources/domain/mod-email/features/email.feature

```diff
@@ -9,27 +9,44 @@ Feature: Email
     Given path 'email'
     When method GET
     Then status 200
+    And match response == { emailEntity: #present, totalRecords: #present }
 
-  @Undefined
-  Scenario: Get email should return 401 if unauthorized error
-    * print 'undefined'
-
-  @Undefined
-  Scenario: Get email should return 404 if email is not found
-    * print 'undefined'
```

Comment:
why have these cases been removed?

Reply:
even if the endpoint doesn't find any email it returns status code 200 with empty result, so this test-case is impassible with current implementation